frontend/src/app/footer/footer.component.html

```diff
@@ -50,6 +50,10 @@ <h3 class="nav-title">Strefa wolontariusza</h3>
               <img class="img-fluid" src="assets/img/logo/logo-wrk.png"
                   alt="Logo WRK - Wielkopolska Rada Koordynacyjna Związek Organizacji Pozarządowych"></a>
         </div>
+        <div class="col-3 ml-4 mr-4 px-0 text-center align-self-center">
+          <a target="_blank" href="https://codeforpoznan.pl/">
+            <img class="img-fluid" src="assets/img/logo/Code_for_Poznan_final_square.png" alt="Logo STX Next"></a>
```

Comment:
`alt="Logo Code for Poznan"` and I don't know if better would be just `alt="Code for Poznan"` instead of `Logo ...`